src/lint/rules/enum-casing.ts

```diff
@@ -0,0 +1,25 @@
+import type { Reporter } from '../algorithm-error-reporter-type';
+import type { OrderedListItemNode } from 'ecmarkdown';
+import { offsetToLineAndColumn } from '../../utils';
+
+const ruleId = 'enum-casing';
+
+/*
+Checks that ~enum-values~ are kebab-cased.
+*/
+export default function (report: Reporter, step: OrderedListItemNode, algorithmSource: string) {
+  for (const item of step.contents) {
+    if (item.name !== 'tilde' || item.contents.length !== 1 || item.contents[0].name !== 'text') {
+      continue;
+    }
+    const text = item.contents[0];
+    if (/[A-Z ]/.test(text.contents)) {
+      const location = offsetToLineAndColumn(algorithmSource, text.location.start.offset);
+      report({
+        ruleId,
+        message: 'enum values should be kebab-cased',
```

Comment:
```suggestion
        message: 'enum values should be lowercase and kebab-cased',
```

Reply:
Wait what? Doesn't kebab-case imply lowercase?

Reply:
`THIS-IS-STILL-KEBAB`